api/src/main/java/org/apache/iceberg/ExpireSnapshots.java

```diff
@@ -96,7 +96,7 @@
    * @param executorService an executor service to parallelize tasks to delete manifests and data files
    * @return this for method chaining
    */
-  ExpireSnapshots executeWith(ExecutorService executorService);
```

Comment:
Can we add the new name in parallel and deprecate this name? We should be careful about changes to the public API and give people at least a release to switch over before removing public methods.

Reply:
Ah I didn't think we hit a release boundary since that addition, sure no problem

Reply:
It seems we did not release it yet.

Reply:
You're right! It is only in master and not 0.9.0: https://github.com/apache/iceberg/commit/ceb6533a593989a86c78277205c0c83108104ded

Nevermind, it should be fine to rename.